### ISSUE TITLE
Crashing on LG V20 Android 7.0

### ISSUE THREAD

**Issue by designhubby** [waterfoul/Gears2PersistantNotifications#2]
The app crashes a lot on the LG v20.  Any idea why?

**Comment by waterfoul** [comment]
Does the S2 watch even work on the LG v20? AFAIK you need to have a samsung device otherwise things won't work right

**Comment by designhubby** [comment]
Yes the Gear S2 and Gear S3, both work on the V20 because Gear Manager now
works on non samsung phones.

It looks like certain apps are causing your app to crash.

For example,when a file is deleted inside FX File Explorer, your app will
crash.

**Comment by waterfoul** [comment]
Ok, I'll add a crash reporter to the app later this week (I'm busy today) and have you reproduce it so I can get more information
